e2e/meta_online_test/test_online_stats_update_4/input.cyp

```diff
@@ -0,0 +1,19 @@
+setup: |-
+  CALL meta.reset();
+
+queries:
+  - |-
+    MERGE (a:Node {id: 0}) MERGE (b:Node {id: 1}) CREATE (a)-[:RELATION]->(b);
+    MERGE (a:Node {id: 1}) MERGE (b:Node {id: 2}) CREATE (a)-[:RELATION]->(b);
+    MERGE (a:Node {id: 2}) MERGE (b:Node {id: 0}) CREATE (a)-[:RELATION]->(b);
+    MERGE (a:Node {id: 3}) MERGE (b:Node {id: 3}) CREATE (a)-[:RELACIJA]->(b);
```

Comment:
please restrict to english language :(, it can be anything you want, but keep it english if it does not signify first name and last name